docs/docs/exclusion.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Exclusion
+sidebar_position: 2
```

Comment:
With the sidebar changes, you no longer have to have the `sidebar_position` item